conf/distro/nilrt.inc

```diff
@@ -74,7 +75,7 @@ VIRTUAL-RUNTIME_base-utils-hwclock = "util-linux-hwclock"
 
 VIRTUAL-RUNTIME_initscripts = "initscripts"
 
-VIRTUAL-RUNTIME_init_manager = "sysvinit"
```

Comment:
Stylistic question: what's the upstream convention for `VIRTUAL-RUNTIME_init_manager` (and `INIT_MANAGER`) with respect to DISTRO_FEATURES? Should both of these settings just key off of `@bb.utils.contains('DISTRO_FEATURES','systemd', ...)`, like so much else does below?